dodola/tests/test_services.py

```diff
@@ -186,9 +186,18 @@ def test_regrid_methods(regrid_method, expected_shape):
     ds_in = grid_global(30, 20)
     ds_in["fakevariable"] = wave_smooth(ds_in["lon"], ds_in["lat"])
 
+    # make fake domain file
+    lon_name = "lon"
+    lat_name = "lat"
+    domain = grid_global(1, 1)
+    domain = domain.rename({"x": lon_name, "y": lat_name})
```

Comment:
Suuuper minor thing but you might not even need this `.rename()` call because you're grabbing unique values below.

(...and the test will fail if we actually did need it...)

Reply:
You're right, we didn't need that!